src/CodeGen_X86.cpp

```diff
@@ -107,6 +107,33 @@ void CodeGen_X86::visit(const Sub *op) {
     }
 }
 
+void CodeGen_X86::visit(const Mul *op) {
+
+#if LLVM_VERSION < 110
```

Comment:
nit: the pattern we use pretty much exclusively in existing code is `LLVM_VERSION >= someconstant`, to make searching for obsolete code easier.

Reply:
In this case there's no code for the case where LLVM_VERSION >= 110, so it would be:
```
#if LLVM_VERSION >= 110
#else 
some code
#endif  
```

Reply:
SGTM